### DISCUSSION TITLE
My fined tuned model only makes noise after 1.5 hours of training

### DISCUSSION THREAD

**Discussion by FacundoMartinezCampos** [coqui-ai/TTS#2003] [General Q&A]
Im currently fine-tuning the model tts_models/es/mai/tacotron2-DDC with a dataset of 115 wavs and after 500 iterations with a batch of 32 im only getting noise from the inference.

## this is my tensorboard: 
![image](https://user-images.githubusercontent.com/54776310/191568256-2a06461a-4987-48b7-ba03-1edc5b5e4ae5.png)

## and this is my train.py

import os

from trainer import Trainer, TrainerArgs

from TTS.tts.configs.glow_tts_config import GlowTTSConfig

from TTS.tts.configs.shared_configs import BaseDatasetConfig, CharactersConfig
from TTS.tts.datasets import load_tts_samples
from TTS.tts.models.glow_tts import GlowTTS
from TTS.tts.utils.text.tokenizer import TTSTokenizer
from TTS.utils.audio import AudioProcessor
if __name__ == '__main__':
        

    output_path = os.path.dirname(os.path.abspath(__file__))

    dataset_config = BaseDatasetConfig(
        formatter="ljspeech",
        meta_file_train="metadata.csv",
        path=os.path.join(output_path, "spanishDataset")
    )

    config = GlowTTSConfig(
        batch_size=32,
        eval_batch_size=16,
        num_loader_workers=4,
        num_eval_loader_workers=4,
        run_eval=True,
        test_delay_epochs=0,
        epochs=1000,
        text_cleaner="phoneme_cleaners",
        use_phonemes=True,
        phoneme_language="es-419",
        phoneme_cache_path=os.path.join(output_path, "phoneme_cache"),
        print_step=25,
        print_eval=True,
        mixed_precision=True,
        output_path=os.path.join(output_path, "output"),
        datasets=[dataset_config],
        save_checkpoints=True,
        save_step=100,
    )
    config.audio.sample_rate = 16000

    ap = AudioProcessor.init_from_config(config)

    tokenizer, config = TTSTokenizer.init_from_config(config)

    train_samples, eval_samples = load_tts_samples(
        dataset_config,
        eval_split=True,
        eval_split_max_size=config.eval_split_max_size,
        eval_split_size=config.eval_split_size,
    )

    model = GlowTTS(config, ap, tokenizer, speaker_manager=None)

    trainer = Trainer(
        TrainerArgs(), config, output_path, model=model, train_samples=train_samples, eval_samples=eval_samples
    )

    trainer.fit()

**Comment by Ahmer-444** [comment]
Facing the same issue, any luck so far?

**Comment by chuchong** [comment]
the same

**Comment by manmay-nakhashi** [comment]
Why are you training tacotron-2 model with glowtts script ? 🤔
